src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/camunda/notification/DefendantClaimDetailsNotificationHandler.java

```diff
@@ -0,0 +1,82 @@
+import uk.gov.hmcts.reform.unspec.callback.CaseEvent;
+import uk.gov.hmcts.reform.unspec.config.properties.notification.NotificationsProperties;
+import uk.gov.hmcts.reform.unspec.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.unspec.model.CaseData;
+import uk.gov.hmcts.reform.unspec.service.NotificationService;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.unspec.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.unspec.callback.CaseEvent.NOTIFY_RESPONDENT_SOLICITOR1_FOR_CLAIM_DETAILS;
+import static uk.gov.hmcts.reform.unspec.helpers.DateFormatHelper.DATE;
+import static uk.gov.hmcts.reform.unspec.helpers.DateFormatHelper.formatLocalDate;
+import static uk.gov.hmcts.reform.unspec.utils.PartyUtils.getPartyNameBasedOnType;
+
+@Service
+@RequiredArgsConstructor
+public class DefendantClaimDetailsNotificationHandler extends CallbackHandler implements NotificationData {
+
+    private static final List<CaseEvent> EVENTS = List.of(NOTIFY_RESPONDENT_SOLICITOR1_FOR_CLAIM_DETAILS);
+    public static final String TASK_ID = "NotifyClaimDetailsRespondentSolicitor1";
+    private static final String REFERENCE_TEMPLATE = "claim-details-respondent-notification-%s";
+
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+    private final CaseDetailsConverter caseDetailsConverter;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return Map.of(
+            callbackKey(ABOUT_TO_SUBMIT), this::notifyRespondentSolicitorForClaimDetails
+        );
+    }
+
+    @Override
+    public String camundaActivityId() {
+        return TASK_ID;
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    private CallbackResponse notifyRespondentSolicitorForClaimDetails(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+
+        notificationService.sendMail(
+            caseData.getRespondentSolicitor1EmailAddress(),
+            notificationsProperties.getRespondentSolicitorClaimDetailsEmailTemplate(),
+            addProperties(caseData),
+            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
+        );
+
+        CaseData updatedCaseData = caseData.toBuilder()
+            .claimDetailsNotificationDate(LocalDate.now())
+            .build();
+
+        return AboutToStartOrSubmitCallbackResponse.builder()
+            .data(caseDetailsConverter.toMap(updatedCaseData))
+            .build();
+    }
+
+    @Override
+    public Map<String, String> addProperties(CaseData caseData) {
+        return Map.of(
+            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+            RESPONDENT_NAME, getPartyNameBasedOnType(caseData.getRespondent1()),
+            ISSUED_ON, formatLocalDate(caseData.getClaimIssuedDate(), DATE)
```

Comment:
Hmm, is this issue date the correct on here? We will have a follow up story for notification to address this.

Reply:
not actually sure, the email wanted the claim issue date, but whether that is when the claim was created in ccd or when the claim was shared I am not sure. I guess this is going to be defined by the date refactor ticket?

Reply:
Yeah we will look into this specific issue as part of another ticket.